projects/extension/Dockerfile

```diff
@@ -51,6 +50,13 @@ WORKDIR /pgai
 COPY . .
 RUN just build install
 
+# Create a custom config file in docker-entrypoint-initdb.d
+RUN mkdir -p /docker-entrypoint-initdb.d && \
+    echo "#!/bin/bash" > /docker-entrypoint-initdb.d/configure-timescaledb.sh && \
+    echo "echo \"shared_preload_libraries = 'timescaledb'\" >> \${PGDATA}/postgresql.conf" >> /docker-entrypoint-initdb.d/configure-timescaledb.sh && \
+    chmod +x /docker-entrypoint-initdb.d/configure-timescaledb.sh
```

Comment:
I ran into issues when running `Create extension timescaledb` and claude recommended this, i'm not quite sure what this does but it works.